src/compile/render-dom/wrappers/Element/Binding.ts

```diff
@@ -52,9 +53,9 @@ export default class BindingWrapper {
 			// we need to ensure that the each block creates a context including
 			// the list and the index, if they're not otherwise referenced
 			const { name } = getObject(this.node.expression.node);
-			const eachBlock = block.contextOwners.get(name);
+			const eachBlock = this.parent.node.scope.getOwner(name);
 
-			eachBlock.hasBinding = true;
+			(<EachBlock>eachBlock).has_binding = true;
```

Comment:
Minor code style nit: supposedly, `as` is the preferred cast method... something, something jsx... and it's a little easier to read.
```suggestion
			(eachBlock as EachBlock).has_binding = true;
```

Reply:
Nice. I switched over all the other occurrences as well